tests/simulations/ringtest/create_data.py

```diff
@@ -0,0 +1,152 @@
+import h5py
+import libsonata
+import numpy as np
+
+
+def make_nodes():
+    name = "RingA"
+    count = 3
+    with h5py.File("nodes_A.h5", "w") as h5:
+        h5.create_dataset(f"/nodes/{name}/0/model_template", data=["hoc:B_BallStick"] * count)
+        h5.create_dataset(f"/nodes/{name}/0/model_type", data=["biophysical"] * count)
+        h5.create_dataset(f"/nodes/{name}/node_type_id", data=[-1] * count)
+
+        # neurodamus/io/cell_readers.py:140: SonataError
+        h5.create_dataset(f"/nodes/{name}/0/mtype", data=["MIKE"] * count)
+
+        # neurodamus/io/cell_readers.py:162: SonataError
+        h5.create_dataset(f"/nodes/{name}/0/x", data=[0.0] * count, dtype=np.float32)
```

Comment:
This looks like it could be a source of optimization; I don't think x/y/z positions are needed other than in the LFP case? So they may be read unnecessarily?

Reply:
I don't see calls to `local_to_global_coord_mapping`, other than in the tests, maybe I am missing something.

Reply:
add a ticket here: https://github.com/openbraininstitute/neurodamus/issues/86